Libraries/Components/Touchable/TouchableOpacity.js

```diff
@@ -215,13 +216,20 @@ class TouchableOpacity extends React.Component<Props, State> {
       ...eventHandlersWithoutBlurAndFocus
     } = this.state.pressability.getEventHandlers();
 
+    const {disabled} = this.props;
+
+    const accessibilityState =
+      disabled != null
+        ? {...this.props.accessibilityState, disabled}
+        : disabled;
```

Comment:
`this.props.accessibilityState` on line 224

Reply:
Also, we can avoid object creation:  
`disable !== this.props.accessibilityState?.disabled`